.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
```

Comment:
I was thinking its easier to manage the basic code formatting this way. Since I didn't find any guideline related to coding style on this project, I thought I add this as a proposal.
If you'd like to keep this off the repository, I can remove it.